library/Fission/Models.hs

```diff
@@ -75,6 +77,14 @@ User
 
   deriving Show Eq
 
+--------------------------------------------------------------------------------
+
+UserChallenge
+  userId    UserId
+  hash      Challenge
+
+  deriving Show Eq
```

Comment:
Are these unique? i.e. can we have multiple active challenges per user?

Reply:
yes they are unique. good call :+1: